packages/theme/src/bootstrap-sass/overrides/_navs.scss

```diff
@@ -263,7 +287,7 @@
     overflow: hidden;
     position: relative;
     width: 100%;
-    height: 43px;
+    // height: 43px;
```

Comment:
Commented?

Reply:
Yes, it's weird, but this is a 1:1 copy from what got into master in `uaf-main` so this will be cleaned up in a future cleanup story.